leaderboardPage.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="./normalize.css"/>
+    <link rel="stylesheet" type="text/css" href="./style.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap" rel="stylesheet">
+    <script src="./script.js" defer></script>
+    <script src="./leaderboardScript.js" defer></script>
+    <title>Surfwave Fury Leaderboard</title>
+</head>
+<body>
+    <main>
+        <div class="leaderboardTopContainer">
+            <div class="leaderboardTitleContainer">
+                <h1 class="leaderboardTitle">Surfwave Fury</h1>
+            </div>
+            <div class="backButtonContainer">
+                <a href="index.html">
+                    <button id="backButton">Back to Surfwave Fury</button>
+                </a>
```

Comment:
i don't like this but i will try remember to pick it up in the next story for morale reasons (not good accessibility)